README.md

```diff
@@ -202,14 +202,24 @@ use the provided Rake task:
 rake dynamo_db:session_store:clean
 ```
 
-## Amazon Simple Email Service (SES) as an ActionMailer Delivery Method
+## ActionMailer delivery with Amazon Simple Email Service
```

Comment:
That's the plan but I wasn't sure whether to do that now or later (trying to avoid confusion) because the other readme says to install the gem which doesn't exist. For now I just duplicated the info and will delete this in one sweep.